packages/background/src/controllers/AccountTrackerController.ts

```diff
@@ -805,20 +832,28 @@ export class AccountTrackerController extends BaseController<AccountTrackerState
                         if (token) {
                             if (
                                 balance.gt(zero) &&
-                                !userTokens.includes(tokenAddress)
+                                !userTokens.includes(tokenAddress) &&
+                                knownTokens.includes(tokenAddress) // the token has to be known (not spam)
```

Comment:
I'm not sure about this condition. There are situation where we don't "know" the token and still it is not an spam. 

For instance, we don't have MATIC on Gnosis Chain (You can manually search it by symbol using the wallet and it won't appear) and it is not an spam. This is the contract address: `0x7122d7661c4564b7C6Cd4878B06766489a6028A2`